src/server/game/Spells/SpellInfo.cpp

```diff
@@ -2133,6 +2133,10 @@ SpellSpecificType SpellInfo::LoadSpellSpecific() const
                         case 8115: // Agility
                         case 8091: // Armor
                             return SPELL_SPECIFIC_SCROLL;
+                        case 12880: // Enrage (Enrage)
+                        case 57518: // Enrage (Wrecking Crew)
+                        case 57514: // Enrage (Imp. Defensive Stance)
+                            return SPELL_SPECIFIC_WARRIOR_ENRAGE;
```

Comment:
This is a confusing change. 
I'm not sure this Warrior logic should be put on generic families. Maybe loading a spell correction and change it's family to SPELLFAMILY_WARRIOR would be wiser for future purposes.
Another thing that's bothering me is that you only added the Rank 1 Enrage spell for **Enrage** (12880) and the Rank 1 enrage for the spell 46867 - **Wrecking Crew (Rank 1)**. Which means that the other talent ranks will not be considered _SPELL_SPECIFIC_WARRIOR_ENRAGE_.

My suggestion would be
1. Finding where does the Modifier for damage is called and see how it's coded (probably on spell file)
2. Adding a priority list on each Enrage effect so one is not overridden by the other whenever procced.

Reply:
1. it is used with GetFirstRankSpell() that means he doesnt need to place every rank there.
2. Spellfamily from those Spell is generic in DBC. Why should he place it in Warrior family
3. I think replacing other proc by new one overridden proc is completely normal. At least it solves the problem

Reply:
You clearly didn't read my test report. But you do you